logging/add-git-id/StackdriverLogWritter.cs

```diff
@@ -0,0 +1,102 @@
+﻿/*
+ * Copyright (c) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using Google.Api;
+using Google.Cloud.Logging.V2;
+using Google.Cloud.Logging.Type;
+using Google.Cloud.DevTools.Source.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Runtime.CompilerServices;
+using System.Reflection;
+using System.Security;
+using Google.Protobuf;
+
+namespace GoogleCloudSamples
+{
+    public static class StackdriverLogWritter
+    {
+        private const string LogId = "YOUR-LOG-ID";
+        private const string ProjectId = "YOUR-PROJECT-ID";
+
+        [MethodImpl(methodImplOptions: MethodImplOptions.NoInlining)]
```

Comment:
I am NOT sure if this is inline , then would the caller information be messed up